examples/kong/kong.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "/kong",
+    "dependencies": [],
+    "instances": 1,
+    "cpus": 1,
+    "mem": 1024,
+    "cmd": "cp $MESOS_SANDBOX/kong.yml /etc/kong/; kong start && tail -f /usr/local/kong/logs/error.log",
+    "uris": [
+      "http://ryanesc.s3.amazonaws.com/samples/kong.yml"
```

Comment:
For now, this is in my personal S3 account. If we like this approach better, we should move it to a more "official" location.